src/css/modules/_ContactDetails.scss

```diff
@@ -50,3 +50,12 @@
     border: none;
   }
 }
+
+.rtl {
```

Comment:
SUPER MINOR but can we stick to coa prefixes -- so .coa-rtl

Reply:
suuuuuure